packages/cli/src/commands/hydrogen/env/list.ts

```diff
@@ -22,9 +22,8 @@ import {
 } from '../../../lib/render-errors.js';
 
 export default class EnvList extends Command {
-  static description = 'List the environments on your Hydrogen storefront.';
-
-  static hidden = true;
+  static description =
```

Comment:
@gfscott I wonder if this would be a good time to review all of the descriptions we have for items? I'll be adding content to `shopify.dev`, too.